targets/TARGET_STM/TARGET_STM32F2/objects.h

```diff
@@ -137,8 +137,9 @@ struct pwmout_s {
 };
 
 struct can_s {
```

Comment:
Hello Laurent,
Could you add #if DEVICE_CAN (in case of future STM32F2 device that does not support CAN) ?